version.php

```diff
@@ -31,4 +31,4 @@
 $plugin->component = 'tool_emailutils';
 $plugin->dependencies = ['local_aws' => 2020061500];
 $plugin->maturity  = MATURITY_STABLE;
-$plugin->supported = [39, 404];
+$plugin->supported = [39, 403];
```

Comment:
can you add the incompatible for 4.4 on this branch please

Reply:
As discussed I've added a hard cutoff for 4.5. I've left 4.4 as limited support.